### PR TITLE
BP-962 Magento 2 - Cancel 'Bank Transfer' orders after due date

### PR DESCRIPTION
**Pull request by vladislav-padalka-hys**

### CODE REVIEW

**Review by saranchuk-hys — APPROVED** [review]

File: Model/Service/Order.php [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade this module to newer
+ * versions in the future. If you wish to customize this module for your
+ * needs please contact support@buckaroo.nl for more information.
+ *
+ * @copyright Copyright (c) Buckaroo B.V.
+ * @license   https://tldrlegal.com/license/mit-license
+ */
+namespace Buckaroo\Magento2\Model\Service;
```
Comment (florinel-chis) [inline Model/Service/Order.php:20]: service classes, in general, should be placed under Service not Model/Service.

File: Model/Service/Order.php [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+    protected function cancelExpiredTransferOrdersPerStore($store)
+    {
+        $this->logging->addDebug(__METHOD__ . '|1|' . var_export($store->getId(), true));
+        $statesConfig = $this->configProviderFactory->get('states');
+        $state = $statesConfig->getOrderStateNew($store);
+        if ($transferConfig = $this->configProviderMethodFactory->get('transfer')) {
+            if ($dueDays = abs($transferConfig->getDueDate())) {
+                $this->logging->addDebug(__METHOD__ . '|5|' . var_export($dueDays, true));
+                $orderCollection = $this->orderFactory->create()->addFieldToSelect(['*']);
+                $orderCollection
```
Comment (florinel-chis) [inline Model/Service/Order.php:81]: manipulating data is part of the Model, which can support a Service class. Easier to read code logic and not get it bloated by composing the collection select and filters

File: Model/Service/Order.php (hunk 1 above)
Comment (vladislav-padalka-hys) [reply Model/Service/Order.php:20]: it's not true
check magento sources
vendor/magento/module-store/Model/Service
vendor/magento/module-sales/Model/Service
also,our module already had Model/Service too

File: Model/Service/Order.php (hunk 2 above)
Comment (vladislav-padalka-hys) [reply Model/Service/Order.php:81]: Magento doesn't limit this and such things can be in repositories, models, services
(example: Magento\Store\Model\Service\StoreConfigManager)
overall, 40 rows method doesn't look like bloated

**Pull request merged**